### DISCUSSION TITLE
Downloading "Highlights" collections

### DISCUSSION THREAD

**Discussion by RaidenDK23** [Avnsx/fansly#137] [Q&A] [answered]
found someone who has a bunch of content not included on the main timeline, but is inside their fansly "Highlight" section.

is there any way to download from that section of a profile?

**Comment by Avnsx** [comment]
Not sure what the highlights section is, would be usefull if you linked a profile that contains such.

**Reply by RaidenDK23** [reply]
under the subscription banner/button, you'll see a bunch of circular buttons with text under each one. all of which lead to a specific highlight collection. those are the ones that dont seem possible to download

**Reply by Avnsx** [reply, marked as answer]
As far as I tried, it isn't possible to download the content from "Highlights" currently.
Same with "Stories", like those 24 hour posts that are like popping up for a short duration and then get deleted.

Those are pretty new features, which this downloader doesn't support yet.

**Reply by Darihx** [reply]
Please add this function in a near future! I've also been encountering so much content just being stored in the Highlights section and there is no way to download it
